bin/burden

```diff
@@ -4104,14 +4029,14 @@ fi
 # Does not return
 #
 if [ $gl_tf_terminate -eq 1 ]; then
-	tf_terminate
+	source $UTILS_DIR/terraform_ops --terminate_list $gl_tf_term_list
 fi
 
 #
 # Does not return
 #
 if [ $gl_tf_list -eq 1 ]; then
-	tf_list
+	source $UTILS_DIR/terraform_ops --tf_list
```

Comment:
Is `source` needed here?

Reply:
Source is required because we want burden to terminate when completed.